### ISSUE TITLE
A new Looking-for/hiring section in the bot page

### ISSUE THREAD

**Issue by ShoXcy** [top-gg/issues#843]
Suggestion

A feature to display need of developers, artists, gfx editors and other custom roles by the bot would be greatly helpful. 
People who love and like the bot and non users, both can easily check if there is a position available where they can contribute and it would make it really easy. 

It can be a simple notifier on the bot page or a full fledged application process.

![image](https://user-images.githubusercontent.com/20400149/143150853-e1a2016e-6003-425f-a9c7-d2cdf3080a98.png)

**Comment by bchurton** [comment]
My view on this is that if a bot dev/team wanted to let people know they were hiring/wanting developers, they should just put it in the description or as an announcement. People come to Top.gg to discover their ideal bot or server, not find a job or business opportunity.

**Comment by TheAlienKnight** [comment]
I see 0 reason for top.gg to implement this. LinkedIn is a thing, indeed is a thing, and discord servers are a thing. If a group is looking to hire, they should use an existing platform, a platform specifically designed for recruitment and job listings. No need or reason to reinvent the wheel.

**Comment by Dev-Voltiac** [comment]
Make it like a banner in side so it would be visible at-least , also please make it big i need to use a microscope to look lol

**Comment by bchurton** [comment]
Like AlienKnight said - why reinvent the wheel? Utilise the existing tools (announcements, descriptions, servers etc) and use services such as LinkedIn or GitHub to promote jobs and contributions

**Comment by ExtremeDevelopments** [comment]
Yeah as stated before, there are sites that are purposed for hiring developers or other jobs you need. There isn't really a point to add a complete section to the top.gg bot pages when you can use the Announcement utility on your bots page if you really want to post it. Users are looking for bots to add to there servers, not for a new project to add to their plate. Definitely should use a announcement or another site for this.
